calculator.go

```diff
@@ -1,13 +1,41 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import (
+	"errors"
+	"math"
+)
+
 // Add takes two numbers and returns the result of adding them together.
 func Add(a, b float64) float64 {
-return a + b
+	return a + b
 }
 
 // Subtract takes two numbers and returns the result of subtracting the second
 // from the first.
 func Subtract(a, b float64) float64 {
-	return b - a
+	return a - b
+}
+
+// Multiply takes two numbers and returns the result of subtracting the second
+// from the first.
+func Multiply(a, b float64) float64 {
+	return a * b
+}
+
+// Divide takes two numbers and returns the result of subtracting the second
+// from the first.
+func Divide(a, b float64) (float64, error) {
+	if b == 0 {
+		return 0, errors.New("division by zero error")
```

Comment:
Good! We could have been even more helpful and included the exact inputs. If you're calling this function a lot, from many different places, it can be hard to nail down the problem call unless you know what the parameters were.